deploy-as-code/helm/charts/digit-works/backend/expense-calculator/values.yaml

```diff
@@ -96,3 +96,19 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: organisation
+  - name: INDIVIDUAL_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: individual
+  - name: MUSTER_ROLL_UPDATE_TOPIC
+    value: "update-musterroll"
+  - name: INDIVIDUAL_UPDATE_TOPIC
+    value: "update-individual-topic"
+  - name: STATE_LEVEL_TENANT_ID
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: egov-state-level-tenant-id
+  - name: EXPENSE_SOR_MIGRATION_MAPPING
```

Comment:
why you did not added if in this?

Reply:
its a must, so if it is not there deployment should actually fail, and not take from application properties